d2l-typography-shared-styles.html

```diff
@@ -52,6 +52,12 @@
 				margin: 1.5rem 0 1.5rem 0;
 			};
 
+			--d2l-label-text {
```

Comment:
I should have mentioned this before, but `font-size` and `line-height` should be relative to the user's base font size -- because the user can change that in their Brightspace settings. The way to make things relative to the base font size is by using `rem` units, or "root em units". The default base size is `20px`, so this should change to:

```
font-size: 0.7rem;
line-height: 1rem;
```

Reply:
Do I need to change anything in `d2l-typography.html`? like `@apply --...` and responsiveness?